lib/params/MapperFunction.cpp

```diff
@@ -372,6 +374,30 @@ vector<double> MapperFunction::getMinMaxMapValue() const {
 }
 
 
+vector<double> MapperFunction::GetCustomMappingSliderRange() const
+{
+    vector<double> defaultv(2, 0);
+    assert(IsUsingCustomMappingSliderRange());
```

Comment:
Should this really be an assert? Use VAssert if so.

Reply:
Done